src/Microsoft.IdentityModel.Clients.ActiveDirectory/Core/WsTrust/WsTrustWebRequestManager.cs

```diff
@@ -141,7 +142,7 @@ public async Task<MexDocument> GetMexDocumentAsync(string federationMetadataUrl,
 
             var uri = new UriBuilder(userRealmUriPrefix + userName + "?api-version=1.0").Uri;
 
-            var httpResponse = await _httpManager.SendGetAsync(uri, null, requestContext).ConfigureAwait(false);
+            var httpResponse = await _httpManager.SendGetAsync(uri, (Dictionary<string, string>)AdalIdHelper.GetAdalIdParameters(), requestContext).ConfigureAwait(false);
```

Comment:
Can you modify `SendGetAsync` to accept an `IDictionary<string, string>` instead of `Dictonary<string, string>` ? It is a trivial change. 

In general methods should take in interfaces to collections and not implementations. In this case, would it matter if I'd passed in a SortedDictionary<String, string> instead of a Dictionary<string, string> ? No - the HttpManager just iterates through it to get the keys and the values out.